package.json

```diff
@@ -12,6 +12,8 @@
     "@salesforce/design-system-react": "^0.10.16",
     "@salesforce/lwc-jest": "^0.6.1",
     "@sentry/browser": "^5.10.2",
+    "chrome": "^0.1.0",
+    "chromedriver": "^78.0.1",
```

Comment:
These should not be here. They are installed using the commands in the Dockerfile, not as node packages.